src/shared/util.js

```diff
@@ -766,21 +766,47 @@ class Util {
 
   // Applies the transform to the rectangle and finds the minimum axially
   // aligned bounding box.
-  static getAxialAlignedBoundingBox(r, m) {
-    const p1 = [r[0], r[1]];
-    Util.applyTransform(p1, m);
-    const p2 = [r[2], r[3]];
-    Util.applyTransform(p2, m);
-    const p3 = [r[0], r[3]];
-    Util.applyTransform(p3, m);
-    const p4 = [r[2], r[1]];
-    Util.applyTransform(p4, m);
-    return [
-      Math.min(p1[0], p2[0], p3[0], p4[0]),
-      Math.min(p1[1], p2[1], p3[1], p4[1]),
-      Math.max(p1[0], p2[0], p3[0], p4[0]),
-      Math.max(p1[1], p2[1], p3[1], p4[1]),
-    ];
+  static getAxialAlignedBoundingBox(rect, transform, output) {
```

Comment:
Given that method no longer returns anything, how about renaming it slightly?
```suggestion
  static axialAlignedBoundingBox(rect, transform, output) {
```